packages/zoe/src/contractSupport/bondingCurves.js

```diff
@@ -98,7 +99,7 @@ export const getOutputPrice = (
   const numerator = BigInt(outputValue) * BigInt(inputReserve) * BIG_10000;
   const denominator =
     (BigInt(outputReserve) - BigInt(outputValue)) * oneMinusFeeScaled;
-  return Nat(Number(numerator / denominator));
+  return Nat(Number(BIG_ONE + numerator / denominator));
```

Comment:
A nitpick, but I think it is easier to read and understand with this order. It also matches the Uniswap implementation here: https://github.com/Uniswap/uniswap-v1/blob/c10c08d81d6114f694baa8bd32f555a40f6264da/contracts/uniswap_exchange.vy#L120

```suggestion
  return Nat(Number(numerator / denominator + BIG_ONE));
```